CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+Version 0.3.1 *(08/02/2017)*
```

Comment:
>To be honest we could remove the date completely and add links to the GH release instead. What you think @ataulm?

I think safer since the date could be incorrect (if no one merges this til tomorrow, then the release doesn't actually happen til the day after).

With the link to releases, the link will be broken, but when the release is made, it will work (or link to releases page)